SCAutolib/__init__.py

```diff
@@ -136,4 +147,21 @@ def set_config(path, value, action="replace", type_=str):
     with open(LIB_CONF, "w") as f:
         yaml.dump(config_data, f)
 
-    env_logger.debug(f"Value for filed {path} is update to {value}")
+    logger.debug(f"Value for filed {path} is update to {value}")
+
+
+def run(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, check=False,
```

Comment:
IMHO there should be at least one liner docstring that could be projected to documentation.
Minor thing - run is not very explanatory name and almost looks like some imported method.

Reply:
And it should look like `subprocess.run` because it's actually the wrapper on it.
And yes, docstring should be there. Will add it